ocaml/typing/typecore.ml

```diff
@@ -239,8 +239,6 @@ exception Error_forward of Location.error
 type in_function =
   { ty_fun: type_expected;
     loc_fun: Location.t;
-    (** [region_locked] is whether the function has its own region. *)
-    region_locked: bool;
   }
```

Comment:
This type is just a tuple `type_expected * Location.t` upstream. Maybe we can change back to that now, to reduce drift?

Reply:
Ahh nice. Will do